docs/data/system/pages.ts

```diff
@@ -63,6 +63,17 @@ const pages = [
     icon: 'CodeIcon',
     children: pagesApi,
   },
+  {
+    pathname: '/system/experimental-api',
+    title: 'Experimental APIs',
+    icon: 'ExperimentIcon',
+    children: [
+      {
+        pathname: '/system/experimental-api/extend-sx-prop',
+        title: 'Extend the `sx` prop',
```

Comment:
```suggestion
        pathname: '/system/experimental-api/sx-config',
        title: 'sx config',
```

How about renaming the page to sx config and grouping these into the page:
- override the default config (from `Override some of the existing behavior `)
- add your own config (from `Extend the sx prop`)
- API

cc @samuelsycamore might have a better name.